packages/graphql-persisted/package.json

```diff
@@ -29,7 +29,8 @@
     "koa-compose": ">=3.0.0 <4.0.0"
   },
   "devDependencies": {
-    "@shopify/sewing-kit-koa": "^6.1.2"
+    "@shopify/sewing-kit-koa": "^6.1.2",
+    "@types/koa": "^2.0.45"
```

Comment:
This library, `@shopify/graphql-persisted`, uses the `Context` from `koa` and would require the consumer to have `koa` installed in their project. There were a number of packages that met this criteria, and I use the following method to resolve them: 1) Add the packages the host is required to have in the `peerDependencies` with a pretty loose version range. and 2) add the types packages used to the `devDependencies`.

Why? I added the `peerDependency` because it is needed yet unused in this library. I added a loose version range to reduce the chances of 2 version of the package been installed as a result of this library.

I added the types in `devDependencies` because they should be there in development, but are not exported in this library. However, there is a middleware `persistedOperationAssociationMiddleware` that uses the `Context` type in one of its arguments and _that_ function _is_ being exported. For this reason, I wonder if the types should be moved to dependencies. If anyone can clarify this, please comment.